### DISCUSSION TITLE
Struggling setting up  HardHat "hh"

### DISCUSSION THREAD

**Discussion by roudra323** [NomicFoundation/hardhat#4971] [Troubleshooting] [answered]
As suggested in the documentation , I ran the command `npm install --global hardhat-shorthand` to install hardhat shorthand. And I installed it successfully. But when I try to run the command `hh test` , idk from where , a kind of html viewer shows up, like this :
![image](https://github.com/NomicFoundation/hardhat/assets/58447036/1cdb7b38-7c80-46d4-8d8a-7960b893bb09)

**My system information:
OS : Windows
hardhat version : "^2.18.2"**

**Comment by kanej** [comment, marked as answer]
Hey @roudra323, I have just run the `hh` cli tool on windows and have reproduced this behaviour. This is clearly a bug, and I have opened an issue for it:

https://github.com/NomicFoundation/hardhat/issues/5040
